docs/source/examples.rst

```diff
@@ -275,124 +275,123 @@ Plotting with Manim
 
 .. manim:: SinAndCosFunctionPlot
     :save_last_frame:
-    :ref_modules: manim.scene.graph_scene
+    :ref_modules: manim.mobject.coordinate_systems
     :ref_classes: MathTex
-    :ref_methods: GraphScene.setup_axes GraphScene.get_graph GraphScene.get_vertical_line_to_graph GraphScene.input_to_graph_point
+    :ref_methods: Axes.get_graph Axes.get_vertical_line_to_graph Axes.input_to_graph_point Axes.get_axis_labels
 
-    class SinAndCosFunctionPlot(GraphScene):
-        def __init__(self, **kwargs):
-            GraphScene.__init__(
-                self,
-                x_min=-10,
-                x_max=10.3,
-                num_graph_anchor_points=100,
-                y_min=-1.5,
-                y_max=1.5,
-                graph_origin=ORIGIN,
-                axes_color=GREEN,
-                x_labeled_nums=range(-10, 12, 2),
-                **kwargs
+    class SinAndCosFunctionPlot(Scene):
+        def construct(self):
+            axes = Axes(
+                x_range=[-10, 10.3, 1],
+                y_range=[-1.5, 1.5, 1],
+                x_length=10,
+                axis_config={"color": GREEN},
+                x_axis_config={
+                    "numbers_to_include": np.arange(-10, 10.01, 2),
+                    "numbers_with_elongated_ticks": np.arange(-10, 10.01, 2),
+                },
+                tips=False,
             )
-            self.function_color = RED
+            axes_labels = axes.get_axis_labels()
+            sin_graph = axes.get_graph(lambda x: np.sin(x), color=BLUE)
+            cos_graph = axes.get_graph(lambda x: np.cos(x), color=RED)
+
+            sin_label = axes.get_graph_label(
+                sin_graph, "\\sin(x)", x_val=-10, direction=UP / 2
+            )
+            cos_label = axes.get_graph_label(cos_graph, label="\\cos(x)")
+
+            vert_line = axes.get_vertical_line(
+                axes.i2gp(TAU, cos_graph), color=YELLOW, line_func=Line
+            )
+            line_label = axes.get_graph_label(cos_graph, "x=2\pi", x_val=TAU, direction=UR)
```

Comment:
Previously, the label `x = 2\pi` wasn't generated with `get_graph_label` -- it simply was a `MathTex` that was moved into place (which explains why it isn't colored in the current version of the documentation).

I don't feel too strongly about it, but have a slight preference for doing it similarly to before (to demonstrate how to get the (scene) coordinates of a point on the graph.

Reply:
I think placing a label next to the curve feels a little clunky (especially since it's basically just doing what `get_graph_label` does under the hood). The colour should probably be reversed back to white to maintain the visuals. If we'd like to show off how to interact with the scene, then maybe using `ax.c2p` would be a better alternative.

Reply:
I'm fine with the solution you have pushed for now. Thanks!